src/Middleware/TrimMiddleware.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Middleware;
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+/**
+ * Request middleware that trims all string values of PUT and POST requests.
+ * Some fields such as passwords are excluded.
+ */
+class TrimMiddleware implements MiddlewareInterface
+{
+    /**
+     * @var List of field names to exclude from trim
```

Comment:
That should be `@var array List of [...]` as the first keyword is expected to be the type

Reply:
[The type is optional](https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/var.html#var). But can be specify that it is a list of strings here.